CreateServer.ps1

```diff
@@ -345,6 +345,9 @@ Function Initialize-Variables {
     $script:Context.TunnelFQDN = "$VmName.$Location.cloudapp.azure.com"
     $script:Context.ServiceFQDN = "$VmName-server.$Location.cloudapp.azure.com"
 
+    $script:Context.TunnelFQDNIpv6 = "$($VmName)IPv6.$Location.cloudapp.azure.com"
```

Comment:
parentheses are not needed here since you aren't grabbing a property of VmName (same for the line below)

Reply:
If we remove the parentheses, we'll have $VmNameIPv6, which isn't a defined variable. I think we need them in this case because we are concatenating a string onto the contents of a variable. There may be a better way to do this.

Reply:
because when we put paranthesis expression in quotes it fails and causes errors, we can update it to

$Context.TunnelFQDNIpv6 = "$VmNameIPv6.$Location.cloudapp.azure.com"
in your PR @LarryFluke :) thank you